src/GroupManager.php

```diff
@@ -93,6 +101,10 @@ public function addGroup($entity_type_id, $bundle_id) {
     $editable->set('groups', $groups);
     $saved = $editable->save();
 
+    // Notify other module we added a new group.
+    // todo: should this be an event?
+    $this->moduleHandler->invokeAll('og_group_created', [$entity_type_id, $bundle_id]);
```

Comment:
I have to somewhat agree with @chx here - from what I can see we actually have two systems in place (and as far as I know there isn't really a rule of thumb when to use which). No doubt the event system is more powerful than the hook, but in terms of DX - hooks are much simpler. 

So unless there are cases we specifically need the event system capabilities, I'd go with hooks.